aws/aws-mcp-cli-commands/src/main/java/software/amazon/smithy/java/aws/mcp/cli/commands/AddAwsServiceBundle.java

```diff
@@ -38,16 +37,23 @@ public class AddAwsServiceBundle extends AbstractAddBundle {
     @Option(names = "--include-write-apis",
             description = "Include write APIs in the MCP server",
             defaultValue = "false")
-    boolean includeWriteApis;
+    Boolean includeWriteApis;
```

Comment:
This won't default to `Boolean.FALSE`?

Reply:
My bad, missed adding that to the commit.